notebooks/official/feature_store/feature_store_streaming_ingestion_sdk.ipynb

```diff
@@ -0,0 +1,772 @@
+{
```

Comment:
<p>we -&gt; you</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1321/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>Done.</p>